content/en/docs/tasks/debug-application-cluster/resource-metrics-pipeline.md

```diff
@@ -1,76 +1,199 @@
 ---
 reviewers:
 - fgrzadkowski
```

Comment:
If we are updating documentation, we should also update reviewers. Both reviewers currently listed haven't contributed in last 3 years.

Reply:
Sounds good to me.

Reply:
@chrismetz09 in my view it's also reasonable to update the page reviewers in a separate, follow-up PR